datahub-graphql-core/src/main/resources/ingestion.graphql

```diff
@@ -133,6 +133,29 @@ type ExecutionRequestResult {
   """
   report: String
 
+  """
+  A structured report for this Execution Request
+  """
+  structuredReport: StructuredReport
+
+}
+
+"""
+A flexible carrier for structured results of an execution request.
+"""
+type StructuredReport {
+  """
+  The type of the structured report. (e.g. INGESTION_REPORT, TEST_CONNECTION_REPORT, etc.)
+  """
+  type: String!
```

Comment:
In the future we may actually want to use an enum here! This doesn't seem to be changing super often

Reply:
totally agreed! yeah once we lock it in I'd like to be more strict on types. it seems to be pretty much there though